tests/utils/test_cache_utils.py

```diff
@@ -916,10 +901,167 @@ def test_hybrid_cache_prompt_longer_than_max_cache_len(self):
             layer_idx=0,
             cache_kwargs={"cache_position": cache_position, "sliding_window": self.window_size},
         )
-        stored_keys, stored_values = self._extract_both_caches(hybrid_cache)
+        k = hybrid_cache.key_cache[0][0, 0, :, 0].tolist()
+        v = hybrid_cache.value_cache[0][0, 0, :, 0].tolist()
+        self.assertEqual(k, values[-self.window_size :], "HybridCache (sliding) did not keep last window tokens")
+        self.assertEqual(v, values[-self.window_size :], "HybridCache (sliding) did not keep last window tokens")
+
+    def test_static_cache_hardcoded(self):
```

Comment:
Yeah, something like this. We can remove the equivalent tests above, and make the docstrings of these tests more complete with step-by-step expected values